takwimu/templates/takwimu/_includes/navbar/_index.html

```diff
@@ -42,12 +42,12 @@
             <a class="dropdown-item" href="#">WIP</a>
           </div>
         </li>
-        <li class="nav-item">
+        <!-- <li class="nav-item">
```

Comment:
Rather than commenting it out, why not just remove it?

Reply:
@kilemensi 
So am still not sure if this needs to be completely removed 
cc @DavidLemayian 
But let me just remove it completely for now

Reply:
@kilemensi done